src/WorkspaceContent.vue

```diff
@@ -75,6 +75,7 @@ export default {
 							color: space.color_code,
 							groupfolderId: space.groupfolder_id,
 							groups: space.groups,
+							added_groups: space.added_groups ?? [],
```

Comment:
Normally, it's not necessary, as you will always return an empty array, no? :thinking:

Reply:
Prefer not do that, as my cs fix is not the same as yours, and it modifies a lot of stuff.